packages/lesswrong/components/common/AnalyticsTracker.jsx

```diff
@@ -0,0 +1,24 @@
+import { registerComponent } from 'meteor/vulcan:core';
+import React, { useCallback } from 'react';
+import { useTracking } from "../../lib/analyticsEvents";
+
+const mouseButtonDecoding = {
+  0: 'main_button',
```

Comment:
I feel like these names are even more confusing than the numbers 0-2 were, and would prefer the traditional (though not pedantically correct) names left, right, middle.

Reply:
Yeah, I see the case for it. Not sure.